### PR TITLE
chore(docs): update docs

### PR DESCRIPTION
**Pull request by sundy-li**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Summary about this PR

ClickHouse does not support clickhouse client, it's just HTTP handler.

Closes #issue

### CODE REVIEW

File: docs/doc/11-integrations/00-api/02-clickhouse-handler.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ description:
 
 ## Overview
 
-Databend is ClickHouse wire protocol-compatible, allow you to connect to Databend server with Clickhouse client, make it easier for users/developers to use Databend.
+Databend is ClickHouse HTTP API wire protocol-compatible, allow you to connect to Databend server with ClickHouse HTTP Handler, make it easier for users/developers to use Databend.
```
Comment (soyeric128) [inline docs/doc/11-integrations/00-api/02-clickhouse-handler.md:12]: Databend is ClickHouse HTTP API wire protocol-compatible, allowing you to connect to Databend with ClickHouse HTTP Handler.

File: docs/doc/00-overview/index.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ import TabItem from '@theme/TabItem';
 <Tabs groupId="whydatabend">
 <TabItem value="Performance" label="Performance">
 
-- Blazing-fast data analytics on object storage.
+- [Blazing-fast data analytics](https://benchmark.clickhouse.com/) on object storage.
```
Comment (soyeric128) [inline docs/doc/00-overview/index.md:21]: [Proved performance by ClickBench](https://benchmark.clickhouse.com/) for data analytics on object storage.

Comment (sundy-li) [reply docs/doc/00-overview/index.md:21]: That's not proven, clickbench is not benched on object storage, I will update it in another.

File: README.md [hunk 3]
```diff
@@ -63,12 +63,12 @@ Databend uses the latest techniques in vectorized query processing to allow you
...
 - __Blazing Performance__
 
-  Databend leverages data-level parallelism(Vectorized Query Execution) and instruction-level parallelism(SIMD) technology, offering blazing performance data analytics.
+  Databend leverages data-level parallelism(Vectorized Query Execution) and instruction-level parallelism(SIMD) technology, offering [blazing performance](https://benchmark.clickhouse.com/) data analytics.
 
 
 - __Git-like MVCC Storage__
 
-  Databend stores data with snapshots. It's easy to query, clone, and restore historical data in tables.
+  [Databend stores data with snapshots](https://databend.rs/blog/databend-engine). Databend can flash back to any history timepoint of data, it's easy to query, clone, and restore historical data in tables.
```
Comment (soyeric128) [inline README.md:71]: Can we use this link in the sentence? https://databend.rs/doc/sql-commands/ddl/table/optimize-table#what-are-snapshot-segment-and-block

And i updated the sentence a little bit:

[Databend uses snapshots to store data](https://databend.rs/doc/sql-commands/ddl/table/optimize-table#what-are-snapshot-segment-and-block), enabling users to effortlessly query, clone, or restore data from any history timepoint.

File: docs/doc/11-integrations/00-api/02-clickhouse-handler.md [hunk 4]
```diff
@@ -2,14 +2,14 @@
...
-  Databend is ClickHouse wire protocol-compatible.
+  Databend is ClickHouse HTTP API wire protocol-compatible.
 ---
 
 ![image](/img/api/api-handler-clickhouse.png)
 
 ## Overview
 
-Databend is ClickHouse wire protocol-compatible, allow you to connect to Databend server with Clickhouse client, make it easier for users/developers to use Databend.
+Databend is ClickHouse HTTP API wire protocol-compatible, allowing you to connect to Databend with ClickHouse HTTP Handler, make it easier for users/developers to use Databend.
```
Comment (soyeric128) [inline docs/doc/11-integrations/00-api/02-clickhouse-handler.md:12]: I think we can get rid of the part "make it easier for users/developers to use Databend.". It makes the whole sentence grammatically incorrect. 
If you need it anyway, we can say:

..., allowing users and developers to easily connect to Databend with ClickHouse HTTP Handler.

**Pull request merged**